src/overview/components/ResultList.jsx

```diff
@@ -27,6 +27,15 @@ const timeGapToSpaceGap = makeNonlinearTransform({
 })
 
 const ResultList = ({searchResult}) => {
+
+    //if no result found display relavant message
+    if(searchResult.rows.length === 0){
+        return <p className={styles.message}>
```

Comment:
This is one of those stylistic choices where I sway back and forth myself. I'd say keep it either way you like now, and we will restyle the whole codebase at some moment if we think it's better. Perhaps combined with a switch to indenting with only two spaces, I just use four because of old Python habits, but it may be less appreciated in the general javascript etiquette.